docs/assets/js/tabs.js

```diff
@@ -0,0 +1,51 @@
+import { Tabs } from 'govuk-frontend';
+
+/**
+ * @param {DOMNode} container - Parent element containing Gov UK tab HTML
+ */
+export function init( container ) {
+  const tabs = container.querySelectorAll( '[data-module="tabs"]' );
+
+  if ( tabs && tabs.length > 0 ) {
+    for ( let i = 0; i < tabs.length; i++ ) {
+      const tab = tabs[i];
+      new Tabs( tab ).init( );
+    }
+  }
+}
+
+/**
+ * This is a helper function that is only used within Netlify CMS.
+ * The original Gov UK tabs code modifies the URL's hash which breaks Netlify
+ * so we replace their functionality with some simple tab switching below.
+ *
+ * @param {DOMNode} tab - Tab element that was clicked
+ * @param {DOMNode} document - Defaults to window.document but overridable for ReactDOM
+ */
+export function changeTab( tab, document = window.document ) {
+  const TABS_CONTAINER_CLASS = '.govuk-tabs';
+  const TAB_CLASS = '.govuk-tabs__list-item';
+  const TAB_CLASS_SELECTED = 'govuk-tabs__list-item--selected';
```

Comment:
Mix of including the leading `.` and not is potentially confusing. I would say leave off the leading `.` in all of them, as that matches how the `classList` API expects class names. Then where you need the period, have `.${ TAB_CLASS }` and where you don't have `TAB_CLASS_SELECTED `.

Reply:
Done in https://github.com/cfpb/design-system/pull/502/commits/698ef41b0fb01e4c36ac8654d3be0c847bca7ec4